lblod/ember-rdfa-editor
# Add option to configure editor as non-editable

### Overview
This PR adds an option to configure the editor as non-editable.
It can be configured in two ways:
- Using the ember `Editor` component, by passing the `editable` argument. The editor will also correctly react to value changes of this argument.
- Using an instance of the `SayController` class, by executing the `toggleEditable` method

### How to test/reproduce
- Start the test-app
- Open the index page
- Test out the `toggle editable` button. This button uses the `Editor` component argument approach
- The `toggleEditable` method of the editor controller should yield the same results.

### Challenges/uncertainties
- The toolbar/sidebar do not react to the editability of the document. At the moment, they should be configured to be enabled/disabled by the user of this addon.
- Unsure if it is useful to keep both pathways to configure the editability of a document (controller vs component argument)

### Checks PR readiness
